Source/GSXib5KeyedUnarchiver.m

```diff
@@ -0,0 +1,3391 @@
+                                           nil];
+          RETAIN(XmlKeyMapTable);
+
+          // These define keys that are always "CONTAINED" since they typically are a combination of key values
+          // stored as separate and/or multiple attributed values that may be combined as in the case of flags
+          // and masks.  There are some that have NO direct cross reference (i.e. NSSupport, NSBGColor, etc)
+          // Each of the ones listed here will MOST PROBABLY have an entry in the 'XmlKeyToDecoderSelectorMap'
+          // below that provides a cross referenced to an asociated decoding method...
+          // If there is an easy way to check whether an existing OLD XIB key is contained within the XIB 5
+          // version the 'containsValueForKey:' method in this file should be modified and the key omitted from this
+          // list (i.e. NSContents, NSAlternateContents, NSIntercellSpacingWidth, NSIntercellSpacingHeight, etc)...
+          XmlKeysDefined = [NSArray arrayWithObjects: @"NSWindowBacking", @"NSWTFlags",
+                                    @"NSvFlags", @"NSBGColor",
+                                    @"NSSize", //@"IBIsSystemFont",
+                                    //@"NSHeaderClipView",
+                                    @"NSHScroller", @"NSVScroller", @"NSsFlags", @"NSsFlags2",
+                                    @"NSColumnAutoresizingStyle", @"NSTvFlags", @"NScvFlags",
+                                    @"NSSupport", @"NSName",
+                                    @"NSMenuItem",
+                                    @"NSDocView",
+                                    @"NSSliderType",
+                                    @"NSCellPrototype", @"NSBrFlags", @"NSNumberOfVisibleColumns",
+                                    @"NSWhite", @"NSRGB", @"NSCYMK",
+                                    //@"NSContents", @"NSAlternateContents",
+                                    @"NSCellFlags", @"NSCellFlags2",
+                                    @"NSButtonFlags", @"NSButtonFlags2",
+                                    @"NSSelectedIndex", @"NSUsesItemFromMenu",
+                                    @"NSNormalImage", @"NSAlternateImage",
+                                    @"NSBorderType", @"NSBoxType", @"NSTitlePosition",
+                                    @"NSTitleCell", @"NSOffsets",
+                                    @"NSMatrixFlags", @"NSNumCols", @"NSNumRows",
+                                    @"NSSharedData", @"NSFlags", @"NSTVFlags", @"NSDefaultParagraphStyle",
+                                    @"NSpiFlags", nil];
+          RETAIN(XmlKeysDefined);
+
+          // These define XML tags (i.e. '<autoresizingMask ...') to an associated decode method...
+          XmlTagToDecoderSelectorMap = [NSDictionary dictionaryWithObjectsAndKeys:
+                                                       @"decodeTableColumnResizingMaskForElement:", @"tableColumnResizingMask",
```

Comment:
Per previous (and below) comment, this really could be indented by four spaces or such compared to the previous line. Aligning to `dictionaryWithObjectsAndKeys:` and then indenting by two additional spaces seems an odd choice.

Reply:
I had to cheat here to get the line better visible. Emacs tried to align the colons within the strings. This really doesn't make sense.